app/src/services/dataService.js

```diff
@@ -58,7 +91,10 @@ module.exports = {
           rejectOnEmpty: true
         }
       );
-      return ipcObj;
+      for (const s of statuses) {
+        s.Notes = await this.getNotes(s.ipcPlanId, s.inspectionStatusId);
+      }
```

Comment:
I can't quite remember if running this as a Promise.all and map would actually make this run any faster/in parallel or not.